sos/policies/redhat.py

```diff
@@ -255,6 +255,9 @@ class RHELPolicy(RedHatPolicy):
 generated in %(tmpdir)s and may be provided to a %(vendor)s \
 support representative.
 """ + disclaimer_text + "%(vendor_text)s\n")
+    upload_url = 'dropbox.redhat.com'
```

Comment:
Maybe could we point to `RH_FTP_HOST` here, to prevent constant string duplication issues?